daq/host.py

```diff
@@ -50,11 +51,22 @@ class MODE:
     MERR = 'merr'
 
 
+# Mapping DHCP test name to host function call
+DHCP_TESTS_MAPPING = {
+    'port_toggle': '_dhcp_port_toggle_test'
```

Comment:
I think this would be better served as a lambda rather than string-converted-to-attribute, e.g., something like https://github.com/grafnu/forch/blob/master/forch/forchestrator.py#L299

It can easily be dynamic (rather than a STATIC_MAPPING) -- performance is not an issue here...

Reply:
i considered that but I wanted to reuse the keys of DHCP_TESTS_MAPPING to generate dhcp_tests() function so you don't have specify twice for the available DHCP tests.